test/errhandling/parallel/try-sync-throws.chpl

```diff
@@ -0,0 +1,16 @@
+// Courtesy of Louis Jenkins via #10805
+// Tracks a non-throwing function using a try around a sync around a
+// coforall+on that calls a throwing function
+module M {
+	proc throwingFn() throws {}
+	proc goodFn() {
+    try! {
+      sync coforall loc in Locales do on loc {
+          throwingFn();
+        }
+		}
```

Comment:
Something is wrong with the spacing here. tabs -> spaces?

Reply:
That's weird.  I copied from Github.  I had to remove the tabs by hand, they didn't get autoreplaced like mine normally do.  Also, they looked like tabs, but backspacing acted like they were spaces ...